ONSdigital/ras-frontstage
# Fix todo page to be more performant

# Motivation and Context
Currently for some respondents on prod, they're unable to load the to-do page because it is hitting the timeout limit. To increase the performance of the to-do page, threads have been added to make multiple calls at once instead of sequentially.
# What has changed
- Added threads to the to-do page to make multiple calls at once and improve the performance

# How to test?
- As a respondent, enroll onto a load of surveys with different collection exercises so you fill up your to-do page on frontstage. The page should be quick to load. From my own testing, its been just over a second.
- Run the acceptance tests
- Just make sure that nothings broken on frontstage, as far as I can see, nothing has changed

# Links
[Trello](https://trello.com/c/aKL0leZQ)

## This can't be merged in until [this PR](https://github.com/ONSdigital/ras-frontstage/pull/427) has been merged in
